tests/unit/nodes/physics_engine/test_fluid_forces.py

```diff
@@ -0,0 +1,107 @@
+
+
+@pytest.mark.parametrize(
+    "apparent_velocity, orientation, expected_angle",
+    [
+        # Tests if apparent velocity is 0
+        (np.array([0, 0]), 0, 0),
+        (np.array([0, 0]), 45, 45),
+        (np.array([0, 0]), 90, 90),
+        (np.array([0, 0]), 180, 180),
+        (np.array([0, 0]), 270, 90),
+        (np.array([0, 0]), 360, 0),
+        (np.array([0, 0]), 450, 90),
+        # Tests if apparent velocity is not 0
+        (np.array([1, 0]), 0, 0),
+        (np.array([0, 1]), 0, 90),
+        (np.array([-1, 0]), 0, 180),
+        (np.array([0, -1]), 0, 90),
+        (np.array([1, 1]), 0, 45),
+        (np.array([1, -1]), 0, 45),
+        (np.array([-1, 1]), 0, 135),
+        (np.array([-1, -1]), 0, 135),
+        # Tests for apparent velocity other than unit vectors
+        (np.array([2, 0]), 0, 0),
+        (np.array([0, 2]), 0, 90),
+        (np.array([-2, 0]), 0, 180),
+        (np.array([0, -2]), 0, 90),
+        (np.array([2, 2]), 0, 45),
+        (np.array([2, -2]), 0, 45),
+        (np.array([-2, 2]), 0, 135),
+        (np.array([-2, -2]), 0, 135),
+        # Tests for orientation other than 0
+        (np.array([1, 0]), 45, 45),
+        (np.array([0, 1]), 45, 45),
+        (np.array([-1, 0]), 45, 135),
+        (np.array([0, -1]), 45, 135),
+        (np.array([1, 1]), 45, 0),
+        (np.array([1, -1]), 45, 90),
+        (np.array([-1, 1]), 45, 90),
+        (np.array([-1, -1]), 45, 180),
+    ],
+)
+def test_calculate_attack_angle(
+    apparent_velocity, orientation, expected_angle, medium_force_setup
+):
+    attack_angle = medium_force_setup.calculate_attack_angle(apparent_velocity, orientation)
+    assert attack_angle == expected_angle
+
+
+@pytest.mark.parametrize(
+    "orientation, expected_lift, expected_drag, apparent_velocity",
+    [
+        # Tests for attack angle 0
+        (0, 0, 140, np.array([44 * math.cos(0), 44 * math.sin(0)])),
+        # # # Tests for attack angle 5
+        (0, 6262, 509, np.array([44 * math.cos(np.deg2rad(5)), 44 * math.sin(np.deg2rad(5))])),
+        # # Tests for attack angle 10
+        (0, 11934, 1568, np.array([44 * math.cos(np.deg2rad(10)), 44 * math.sin(np.deg2rad(10))])),
+        # # Tests for attack angle 15
+        (0, 15162, 3035, np.array([44 * math.cos(np.deg2rad(15)), 44 * math.sin(np.deg2rad(15))])),
+        # Tests for attack angle 20
+        (
+            0,
+            11768,
+            3249,
+            np.array([44 * math.cos(np.deg2rad(20)), -44 * math.sin(np.deg2rad(20))]),
+        ),
+    ],
+)
+def test_compute_forces(
+    medium_force_setup, orientation, expected_lift, expected_drag, apparent_velocity
+):
+    lift_force, drag_force = medium_force_setup.compute(apparent_velocity, orientation)
+
+    def calculate_magnitude(force):
+        return np.sqrt(force[0] ** 2 + force[1] ** 2)
+
+    print(lift_force, drag_force)
+    print(calculate_magnitude(lift_force), calculate_magnitude(drag_force))
```

Comment:
When you're finished debugging, any print statements should be removed from test functions

Reply:
will do